Library/Homebrew/language/node.rb

```diff
@@ -16,6 +16,18 @@ def self.pack_for_installation
       # fed to `npm install` only symlinks are created linking back to that
       # directory, consequently breaking that assumption. We require a tarball
       # because npm install creates a "real" installation when fed a tarball.
+      if (package = Pathname("package.json")) && package.exist?
+        begin
+          pkg_json = JSON.parse(package.read)
+        rescue JSON::ParserError
+          $stderr.puts "Could not parse package.json"
+          raise
+        end
+        removed = nil
+        removed ||= pkg_json["scripts"]&.delete("prepare")
+        removed ||= pkg_json["scripts"]&.delete("prepack")
+        package.write("package.json", JSON.pretty_generate(pkg_json)) if removed
```

Comment:
```suggestion
        package.write(JSON.pretty_generate(pkg_json)) if removed
```

As it's going to write to the filename by default. That this didn't cause a CI failure means a test might be nice here.